recipe/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "streamlit-condition-tree" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cedricvlt/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5895a9c0546cf5facc9db0f84d76310403c22da291c99ec0519235d12c5e480b
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [py<38 or s390x]
```

```suggestion
  skip: true  # [py<37 or s390x]
```
https://github.com/cedricvlt/streamlit-condition-tree/blob/e1eccc29f407fbd768405d26ebfedf59a476631e/setup.py#L20